docs/conf.py

```diff
@@ -0,0 +1,72 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
Please remove all auto-generated comments and move any helpful links to the PR description.

Reply:
I would keep the auto-generated comments, they are concise and provide useful links that may help newcomers wanting to start developing. 

Also, I am a very much in favour of links directly in source code files where it make sense. There is no point putting links at another place if the only time you might need it is when looking at a specific file, it would just create more friction by requiring the user to search the links by him/her-self.

My point is: keeping comments in these files costs us basically nothing, but might save a few minutes the next time we need to edit these files, and might reduce the friction for a newcomer to contribute. All in all, a win.